src/app/core/utils/translate/pwa-translate-compiler.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable } from '@angular/core';
+import { TranslateCompiler } from '@ngx-translate/core';
+
+import { Translations } from 'ish-core/internationalization.module';
+
+@Injectable()
+export class PWATranslateCompiler implements TranslateCompiler {
+  private static PLURAL_REGEX = /(.*)\{\{\s*(\w+)\s*,\s*(plural|select)\s*,\s*(.*?\})\s*\}\}(.*)/s;
```

Comment:
Looks like your changes work well, but could you please leave explanatory comments so everyone can understand these regexes and the entire compiler more easily? ;)

Reply:
✔️